exec/cnex/framestack.c

```diff
@@ -18,6 +18,7 @@ TFrameStack *framestack_createFrameStack(uint32_t capacity)
     frames->capacity = capacity;
     frames->top = -1;
     frames->max = -1;
+    frames->stack_depth = -1;
```

Comment:
I don't understand what this is doing here. The `stack_depth` element should be a member of each frame, not a member of the frame stack itself.

Reply:
Fixed.  Removed from the FrameStack structure.